coverage_comment/main.py

```diff
@@ -162,10 +162,10 @@ def generate_comment(
             filename=config.COMMENT_FILENAME,
             content=comment,
         )
-        github.set_output(COMMENT_FILE_WRITTEN=True)
+        github.set_output(config, COMMENT_FILE_WRITTEN=True)
         log.debug("Comment stored locally on disk")
     else:
-        github.set_output(COMMENT_FILE_WRITTEN=False)
+        github.set_output(config, COMMENT_FILE_WRITTEN=False)
```

Comment:
just a note: I see you pass config by position rather than by name (`config=config`).
On functions that have multiple parameters, I almost always use named arguments.

I guess one thing you might or might not be wondering is whether this might interfere with this function defining `**kwargs`, so just to be extra-clear: `**kwargs` contains all the keyword arguments that were not bound to "normal" arguments, so you can use a keyword arg here.